test/distributed/_tensor/test_random_ops.py

```diff
@@ -72,6 +72,9 @@ def test_init_ops(self):
         )
         self._run_init_op(torch.nn.init.normal_, mean=1.5, std=0.8)
         self._run_init_op(torch.nn.init.uniform_, a=0, b=1.2)
+        self._run_init_op(torch.rand_like)
+        self._run_init_op(torch.randn_like)
+        self._run_init_op(torch.randint_like, low=0, high=100)
```

Comment:
Upstream might want these to be in it's own test because they're not in the `nn.init` module

Reply:
Yep agreed. I might have to rename it. Lets see